src/Concerns/MakesAssertions.php

```diff
@@ -215,11 +215,14 @@ public function assertDontSeeLink($link)
      */
     public function seeLink($link)
     {
-        $selector = trim($this->resolver->format("a:contains('{$link}')"));
+        $selector = trim($this->resolver->format("a[href*='{$link}']"));
```

Comment:
that is true, one solution would be to check the innerHTML of the element. But I think another solution would be to split this one into two seperate methods `seeLinkText` and `seeLinkHref` @taylorotwell what do you think?